scvi/data/_preprocessing.py

```diff
@@ -449,6 +449,39 @@ def add_dna_sequence(
         assert len(seqs) == len(chrom_df)
         seq_dfs.append(pd.DataFrame(seqs, index=chrom_df.index))
 
-    sequence_df = pd.concat(seq_dfs, axis=0).loc[adata.var_names]
-    adata.varm[sequence_varm_key] = sequence_df
-    adata.varm[code_varm_key] = sequence_df.applymap(_dna_to_code)
+    output_df = pd.concat(output_dfs, axis=0).loc[adata.var_names]
```

Comment:
```suggestion
    output_df = pd.concat(seq_dfs, axis=0).loc[adata.var_names]
```

Is this what was meant?